packages/playground-examples/copy/ja/TypeScript/Primitives/Unknown and Never.ts

```diff
@@ -17,75 +17,73 @@ const myAccount = jsonParser(`{ "name": "Dorothea" }`);
 myAccount.name;
 myAccount.email;
 
-// If you hover on jsonParser, you can see that it has the
-// return type of any, so then does myAccount. It's possible
-// to fix this with Generics - but it's also possible to fix
-// this with unknown.
+// 変数jsonParserにカーソルを合わせると、戻り値の型がany型であることがわかります。
+// そして、変数myAccountも同様にany型になります。
+// こちらについて、ジェネリクスを使って型付けすることもできますが、
+// unknown型を使うことによっても修正できます。
 
 const jsonParserUnknown = (jsonString: string): unknown => JSON.parse(jsonString);
 
 const myOtherAccount = jsonParserUnknown(`{ "name": "Samuel" }`);
 
 myOtherAccount.name;
 
-// The object myOtherAccount cannot be used until the type has
-// been declared to TypeScript. This can be used to ensure
-// that API consumers think about their typing up-front:
+// myOtherAccountの型をTypeScriptに示すまでは、myOtherAccountのプロパティを使用する
+// ことができません。こうすることによって、APIの使用者に、型について前もって
+// 考えさせることができます。
 
 type User = { name: string };
 const myUserAccount = jsonParserUnknown(`{ "name": "Samuel" }`) as User;
 myUserAccount.name;
 
-// Unknown is a great tool, to understand it more read these:
+// unknown型はとても優れたツールです。
+// unknown型をより深く理解したい場合はこれらを参照してください。
 // https://mariusschulz.com/blog/the-unknown-type-in-typescript
 // https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-0.html#new-unknown-top-type
 
-// Never
+// never型
 
-// Because TypeScript supports code flow analysis, the language
-// needs to be able to represent when code logically cannot
-// happen. For example, this function cannot return:
+// TypeScriptは、データフローの解析をサポートしているため、
+// 論理的に絶対に発生しないコードを表現することができる必要があります。
+// 例えば、こちらの関数は絶対にreturnされません。
 
 const neverReturns = () => {
-  // If it throws on the first line
-  throw new Error("Always throws, never returns");
+  // 1行目で例外を投げる場合
+  throw new Error("必ず例外が投げられるので、戻り値は返りません");
 };
 
-// If you hover on the type, you see it is a () => never
-// which means it should never happen. These can still be
-// passed around like other values:
+// この関数にカーソルを合わせると、型が () => neverとなっていることがわかります。
+// つまり、この関数は絶対にreturnされません。
+// neverは、他の値と同様に、代入することは可能です。
```

Comment:
IMO: 直訳気味にした方がよいかと思いました。
```suggestion
// これらは、他の値と同様に、代入することは可能です。
```